### PR TITLE
Update title of "Search Rubygems" extension

### PR DESCRIPTION
**Pull request by aleks**
## Description

After talking to @pernielsentikaer a while ago in slack, he notified me that the name of my [Search Rubygems extension](https://github.com/raycast/extensions/pull/501), should have the suggested name seen in this PR.

Initial extension submission PR can be found here: https://github.com/raycast/extensions/pull/501

## Type of change

- Bug fix / improvement for my extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by mattisssa** [comment]
Do you think contain `Rubygems` keyword in the subtitle and in the title of command is the good idea?

**Comment by aleks** [comment]
@mattisssa i got this change request from @pernielsentikaer via slack. The initial version (before the title change request, in my initial PR, looked like this https://github.com/raycast/extensions/blob/df0820649be811e553814064319e933dcb8f4540/extensions/search-rubygems/package.json)

From my point of view, there is no need for a subtitle, but since @pernielsentikaer only suggested the title change, i expected that this was some kind of convention (even though i didn't see this in the docs). I'm totally fine with removing the subtitle completely. :)

File: extensions/search-rubygems/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "search-rubygems",
-  "title": "Search Documentation",
+  "title": "Search Rubygems",
   "subtitle": "RubyGems",
```
Comment (pernielsentikaer) [inline extensions/search-rubygems/package.json:5]:
```suggestion
```
There is no subtitle in the root, so yes, it should be removed 😄

**Comment by pernielsentikaer** [comment]
It was my fault, last review I thought that it was inside a command 😊

**Comment by aleks** [comment]
@pernielsentikaer done! :)

**Review by pernielsentikaer — APPROVED** [review]
I think everything is as expected now, thanks 😄

**Review by mathieudutour — APPROVED** [review]

**Pull request merged**